### PR TITLE
#9: Added file deletion when deleting an entity

### PR DESCRIPTION
**Pull request by ulin-evgeny**

### CODE REVIEW

**Review by DenTray — COMMENTED** [review]
please cover new come by autotests

File: tests/MediaTest.php [hunk 1]
```diff
@@ -98,11 +98,19 @@ public function testBulkCreate(): void
 
     public function testDelete(): void
     {
+        Storage::fake();
+
+        $filePath = 'Photo';
+        Storage::put($filePath, 'content');
+        Storage::assertExists($filePath);
```
Comment (DenTray) [inline tests/MediaTest.php:105]:
```suggestion
```

File: tests/MediaTest.php [hunk 2]
```diff
@@ -98,11 +98,19 @@ public function testBulkCreate(): void
 
     public function testDelete(): void
     {
+        Storage::fake();
```
Comment (DenTray) [inline tests/MediaTest.php:101]: let's move faker to setup method

**Review by DenTray — CHANGES_REQUESTED** [review]

File: tests/MediaTest.php (hunk 2 above)
Comment (ulin-evgeny) [reply tests/MediaTest.php]: Done.

File: tests/MediaTest.php (hunk 1 above)
Comment (ulin-evgeny) [reply tests/MediaTest.php]: Done.

**Review by DenTray — APPROVED** [review]

**Pull request merged**